graph-builder/src/graph.rs

```diff
@@ -108,8 +109,11 @@ pub async fn index(
     let path = req.uri().path();
     GRAPH_INCOMING_REQS.with_label_values(&[path]).inc();
 
+    let generic_header: Vec<actix_web::http::HeaderValue> =
+        vec![header::HeaderValue::from_static(CONTENT_TYPE)];
+
     // Check that the client can accept JSON media type.
-    commons::validate_content_type(req.headers(), CONTENT_TYPE)?;
+    let _version: String = commons::validate_content_type(req.headers(), generic_header)?;
```

Comment:
`content_type` is probably a better name for this variable, since it's holding `application/json` or other supported media type string.